### ISSUE TITLE
Show "Themes created" on the user profile page

### ISSUE THREAD

**Issue by tofumatt** [mozilla/addons-frontend#4281]
Create a card that shows themes a user has created on the profile.

<img width="482" alt="screenshot 2018-01-31 14 53 44" src="https://user-images.githubusercontent.com/90871/35629455-8db3d58a-0696-11e8-91f3-8fe0bb5aa808.png">

**Comment by ioanarusiczki** [comment]
@tofumatt I've been doing some testing on AMO dev with FF59 (Win10).

I have three questions:

1. The  "More Themes by DevTesting" is displayed in both cases: user view and public view. I don't know if the text "Themes I've created" is expected to be displayed as in the card displayed above and also as it is mentioned in the tracker https://github.com/mozilla/addons-frontend/issues/2749 
2. A number of six themes appears which is fine but only after hovering over the theme the rating stars are shown.  I don't know if that is as expected or if the ratings and number of users should be displayed simultaneously as above.
3. Add to button is not available yet but I guess this is an upcoming implementation.
4. I filed https://github.com/mozilla/addons-frontend/issues/4796 and https://github.com/mozilla/addons-frontend/issues/4797 for what I've noticed while testing.
